BatchAuctionOptimization/batchauctions.tex

```diff
@@ -283,12 +265,12 @@ \section{Problem statement}
 \begin{itemize}
   \item \emph{(limit price)}:\\
   for all orders
-  $ \omega_i = (j_i,k_i,\overline{x}_i,\overline{y}_i,\pi_i) \in \orders $:
-  the order can only be executed (fully or fractionally) if the exchange rate does not exceed the
-  limit price, \ie $ p_{j_i|k_i} \le \pi $.
+  $ \omega = (j,k,\overline{y},\pi) \in \orders $:
+  the order can only be executed (fully or fractionally) if the exchange rate satisfies the
```

Comment:
This means "≤ or ≥ depending on context", before it was "≤" which seems less ambiguous.

Reply:
I don't see any inequalities here.